frontend/src/components/ProjectMemberPanel.vue

```diff
@@ -206,14 +280,75 @@ export default defineComponent({
       state.error = "";
     };
 
+    const syncFromVCS = () => {
+      // update project's role provider
+      const projectPatch: ProjectPatch = { roleProvider: "GITLAB_SELF_HOST" };
+
+      const promiseList = [];
+
+      promiseList.push(
+        store.dispatch("project/syncMemberRoleFromVCS", {
+          projectId: props.project.id,
+        })
+      );
+
+      promiseList.push(
```

Comment:
This is a member sync. Why do we need to patch project?

Reply:
We need to update the role provider field of that project.